graviti/portex/template.py

```diff
@@ -190,7 +190,7 @@ def template(
     type_: Type[PortexExternalType] = type(
         name,
         (PortexExternalType,),
-        {"parameters": params, "dependences": factory.dependences, "factory": factory},
+        {"params": params, "dependences": factory.dependences, "factory": factory},
```

Comment:
what is this?

Reply:
This is the attr of `PortexExternalType`, which should not be the key `parameters` in yaml.